gsy_framework/data_classes.py

```diff
@@ -350,60 +350,60 @@ def csv_fields(cls) -> Tuple:
 
     def csv_values(self) -> Tuple:
         """Return values of class members that are needed for creation of CSV export."""
-        rate = round(self.offer_bid.energy_rate, 4)
-        return self.creation_time, rate, self.offer_bid.energy, self.seller, self.buyer
+        rate = round(self.order.energy_rate, 4)
```

Comment:
```suggestion
        rate = limit_float_precision(self.order.energy_rate)
```
Better to use a common helper method.